src/Symfony/Component/Security/Http/Tests/Authentication/DefaultAuthenticationFailureHandlerTest.php

```diff
@@ -47,7 +48,7 @@ public function testForward()
             ->method('createRequest')->with($this->request, '/login')
             ->will($this->returnValue($subRequest));
 
-        $response = $this->getMock('Symfony\Component\HttpFoundation\Response');
+        $response = new Response();
```

Comment:
Tests are failing without it (see https://ci.appveyor.com/project/fabpot/symfony/build/1.0.12032) but i'll remove them as soon as https://github.com/symfony/symfony/issues/19736 is merged.